code/src/components/Age.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+
+const ageGroups = ['0-19','20-29','30+'] 
+
+export const AgeRadioOption = ({ageGroupSpan, setAgeGroup}) => {
+  
+  return (
+  <div className='age-input'>
+    <p>Age</p>
+      {ageGroups.map((age) => (
+        <label htmlFor='age' key={age}>
```

Comment:
Since you’re wrapping the input tag in a label tag, you don’t need a htmlFor in the label tag. I suggest you remove this from the code since it’s not necessary 🙂